JLRoutes/JLRoutes.h

```diff
@@ -38,6 +38,12 @@ static NSString *const kJLRoutesGlobalNamespaceKey = @"JLRoutesGlobalNamespace";
 /// Registers a routePattern with default priority (0) in the receiving scheme namespace.
 + (void)addRoute:(NSString *)routePattern handler:(BOOL (^)(NSDictionary *parameters))handlerBlock;
 - (void)addRoute:(NSString *)routePattern handler:(BOOL (^)(NSDictionary *parameters))handlerBlock; // instance method
++ (void)addRoute:(NSString *)routePattern
+       validator:(BOOL (^)(NSDictionary *parameters))validatorBlock
```

Comment:
It'd probably make sense to do it... The only reason I hadn't was that it already existed that way, which isn't a good reason.